package.json

```diff
@@ -15,7 +15,7 @@
     "coverage": "jest --coverage"
   },
   "dependencies": {
-    "@nypl/design-system-react-components": "3.1.1",
+    "@nypl/design-system-react-components": "3.1.1-modal-rc-3",
```

Comment:
@7emansell DS 3.1.3 was just release and it contains the latest `Modal` updates. Thanks for working on it!